lib/core/api.js

```diff
@@ -723,11 +729,15 @@ api.errors.log = function(error, message, properties) {
   api.log('GET /errors');
 
   if (rollbar) {
-    if (error.data && error.data.blobId) {
-      rollbar.error(error, { blobId: error.data.blobId });
-    } else {
-      rollbar.error(error);
+    var extra = {};
```

Comment:
@gniezen - I reorganized most of this. Could use another glance.

Reply:
Looks good! :+1: